actionlist/app/containers/ActionListContainer.js

```diff
@@ -1,17 +1,69 @@
-import React from 'react';
-
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import {
   View,
+  DrawerLayoutAndroid,
 } from 'react-native';
 
 import Header from '../components/header';
 import DisplayList from '../components/display_list';
+import SideMenu from '../components/SideMenu';
+import { toggleDrawer } from '../actions/action_items';
+
+class ActionListContainer extends Component {
```

Comment:
Have you tried this approach: https://stackoverflow.com/questions/42985775/using-drawerlayoutandroid-with-redux so you can keep state out of this component?